newrelic/resource_newrelic_infra_alert_condition.go

```diff
@@ -140,9 +140,8 @@ func resourceNewRelicInfraAlertCondition() *schema.Resource {
 	}
 }
 
-func buildInfraAlertConditionStruct(d *schema.ResourceData) (*newrelic.AlertInfraCondition, error) {
-
-	condition := newrelic.AlertInfraCondition{
+func buildInfraAlertConditionStruct(d *schema.ResourceData) (*alerts.InfrastructureCondition, error) {
```

Comment:
consider renaming to `expandInfraAlertCondition`?

Reply:
can do